playground/src/components/selectors/experiment_submissions_select.tsx

```diff
@@ -0,0 +1,351 @@
+import type { Submission } from "@/model/submission";
+import type { Exercise } from "@/model/exercise";
+
+import { useState } from "react";
+
+import useSubmissions from "@/hooks/playground/submissions";
+import useFeedbacks from "@/hooks/playground/feedbacks";
+import SubmissionList from "@/components/submission_list";
+
+type ExperimentSubmissionsSelectProps = {
+  disabled?: boolean;
+  exercise?: Exercise;
+  trainingSubmissions?: Submission[];
+  evaluationSubmissions?: Submission[];
+  onChangeTrainingSubmissions: (submissions: Submission[] | undefined) => void;
+  onChangeEvaluationSubmissions: (submissions: Submission[]) => void;
+};
+
+export default function ExperimentSubmissionsSelect({
+  disabled,
+  exercise,
+  trainingSubmissions,
+  evaluationSubmissions,
+  onChangeTrainingSubmissions,
+  onChangeEvaluationSubmissions,
+}: ExperimentSubmissionsSelectProps) {
+  const { data, error, isLoading } = useSubmissions(exercise);
+  const { data: feedbacks } = useFeedbacks(exercise);
+
+  const [sumbissionsSelectType, setSumbissionsSelectType] = useState<
+    "random" | "next"
+  >("random");
```

Comment:
This looks a bit weird. Is it because of prettier?
```suggestion
  const [sumbissionsSelectType, setSumbissionsSelectType] = useState<"random" | "next">("random");
```

Reply:
Yeah, I think we need configure this better. For my taste it is breaking too early.